src/helpers/muxedAccount.ts

```diff
@@ -0,0 +1,62 @@
+import { MuxedAccountFieldType } from "@/types/types";
+
+import { Account, MuxedAccount } from "stellar-sdk";
+
+export const muxedAccount = {
+  generate: ({
+    baseAddress,
+    muxedAccountId,
+  }: {
+    baseAddress: string;
+    muxedAccountId: string;
+  }): Partial<MuxedAccountFieldType> => {
+    try {
+      const muxedAccount = new MuxedAccount(
+        new Account(baseAddress, "0"),
+        muxedAccountId,
+      );
+      return {
+        muxedAddress: muxedAccount.accountId(),
+        error: "",
+      };
+    } catch (e: any) {
+      return {
+        muxedAddress: "",
+        error: `Something went wrong. ${e.toString()}`,
+      };
+    }
```

Comment:
If we want to return the same object in both cases, we could optimize the code a bit.

```suggestion
    let muxedAddress = "";
    let error = "";
    
    try {
      const muxedAccount = new MuxedAccount(
        new Account(baseAddress, "0"),
        muxedAccountId,
      );
      
      muxedAddress = muxedAccount.accountId();
    } catch (e: any) {
      error = `Something went wrong. ${e.toString()}`;
    }
    
    return { muxedAddress, error }
```